broker-daemon/proto/broker.proto

```diff
@@ -360,16 +360,43 @@ message Order {
     CANCELLED = 6;
   }
 
+  message Date {
```

Comment:
I'm pretty sure this name will conflict with the one for Fills. And in any event, it's not very descriptive. Perhaps `OrderDates` ? or `OrderStateDates`?